src/core/lib/event_engine/posix_engine/posix_engine.cc

```diff
@@ -77,6 +79,19 @@ using namespace std::chrono_literals;
 namespace grpc_event_engine {
 namespace experimental {
 
+namespace {
+
+grpc_core::NoDestruct<ObjectGroupForkHandler> g_timer_fork_manager;
+
+class Capture {
```

Comment:
nit: prefer naming this something fork-related, like `TimerForkCallbackMethods`. Seeing `Capture` in the middle of the posix engine code is a bit uninformative.

Reply:
Done.